<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-JC2HQ9T" ignored="0" name="Test" tests="6" failures="1" timestamp="11 Mar 2020 19:18:14 GMT" time="56.212" errors="0">
  <testcase name="AddVariants" time="0.015" classname="com.test.LoginTest"/>
  <testcase name="AddVariants" time="0.0" classname="com.test.LoginTest"/>
  <testcase name="AddVariants" time="0.003" classname="com.test.LoginTest"/>
  <testcase name="AddVariants" time="0.0" classname="com.test.LoginTest"/>
  <testcase name="checkErrorMessageForEmail" time="0.811" classname="com.test.LoginTest"/>
  <testcase name="checkErrorMessageForPassword" time="1.256" classname="com.test.LoginTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Please enters password 123456] but found [Please enter password 123456]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Please enters password 123456] but found [Please enter password 123456]
at com.Util.Utility.assertAll(Utility.java:79)
at com.test.LoginTest.checkErrorMessageForPassword(LoginTest.java:64)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- checkErrorMessageForPassword -->
</testsuite> <!-- Test -->
